src/shared_modules/utils/rocksDBWrapper.hpp

```diff
@@ -482,6 +483,54 @@ namespace Utils
             }
         }
 
+        /**
+         * @brief Delete all key-value pairs from the database.
+         *
+         * This method deletes all key-value pairs stored in the database. It iterates through all family columns
+         * and uses a provided callback function to handle each deleted key. After deletion, it commits the changes
+         * to the database.
+         *
+         * @param callback A callback function that takes a string reference representing the deleted key.
+         *
+         * @throws std::runtime_error if an error occurs during data deletion.
+         */
+        void deleteAll(std::function<void(std::string&,std::string&)> callback)
```

Comment:
The parameter 'callback' is copied for each invocation but only used as a const reference; consider making it a const reference

Reply:
Done! Thx!